src/main/scala/lms/collection/ArrayOps.scala

```diff
@@ -10,28 +10,43 @@ import lms.macros.SourceContext
 
 trait ArrayOps { b: Base =>
 
-  def NewArray[T:Manifest](x: Rep[Int]): Rep[Array[T]] = {
+  def NewArray[T: Manifest](x: Rep[Int]): Rep[Array[T]] = {
     Wrap[Array[T]](Adapter.g.reflectMutable("NewArray", Unwrap(x)))
   }
-  def Array[T:Manifest](xs: Rep[T]*): Rep[Array[T]] = {
-    Wrap[Array[T]](Adapter.g.reflectMutable("Array", xs.map(Unwrap(_)):_*))
+  def Array[T: Manifest](xs: Rep[T]*): Rep[Array[T]] = {
+    Wrap[Array[T]](Adapter.g.reflectMutable("Array", xs.map(Unwrap(_)): _*))
   }
-  implicit class ArrayOps[A:Manifest](x: Rep[Array[A]]) {
-    def apply(i: Rep[Int]): Rep[A] = x match {
-      case Wrap(_) => Wrap[A](Adapter.g.reflectRead("array_get", Unwrap(x), Unwrap(i))(Unwrap(x)))
-      case EffectView(x, base) => Wrap[A](Adapter.g.reflectRead("array_get", Unwrap(x), Unwrap(i))(Unwrap(base)))
-    }
-    def update(i: Rep[Int], y: Rep[A]): Unit = x match {
-      case Wrap(_) => Adapter.g.reflectWrite("array_set", Unwrap(x), Unwrap(i), Unwrap(y))(Unwrap(x))
-      case EffectView(x, base) => Adapter.g.reflectWrite("array_set", Unwrap(x), Unwrap(i), Unwrap(y))(Unwrap(base))
-    }
+  implicit class ArrayOps[A: Manifest](x: Rep[Array[A]]) {
+    def apply(i: Rep[Int]): Rep[A] =
+      x match {
```

Comment:
prefer the old format

Reply:
do you mean `x match {` should be on the same line of `def ...`?

Reply:
yeah, I tried to fix it myself but didn't got too much time to read the documentation for formatting

Reply:
I agree - the `x match {` should be on the same line